scripts/docker-run.sh

```diff
@@ -42,4 +43,4 @@ docker run -i -v "${SOF_TOP}":/home/sof/work/sof.git \
 	--env https_proxy="$https_proxy" \
 	--user "$(id -u)" \
 	$SOF_DOCKER_RUN \
-	thesofproject/sof "$@"
+	sof "$@"
```

Comment:
Our documentation used to recommend creating such a local `sof` tag but I removed it after wasting too much time forgetting to update it.